src/body.js

```diff
@@ -483,6 +490,10 @@ export function writeToStream(dest, instance) {
 		// body is buffer
 		dest.write(body);
 		dest.end()
+	} else if (isArrayBuffer(body)) {
+		// body is array buffer
+		dest.write(typedarrayToBuffer(body));
```

Comment:
Why not just `Buffer.from(body)`? Plus, `body` isn't exactly a TypedArray here, as the function call seems to imply.

Reply:
also, why not use `ArrayBuffer.isView(body)` instead of `is-array-buffer`?

Reply:
string-to-arraybuffer is only used in the testing server.

> also, why not use `ArrayBuffer.isView(body)` instead of `isArrayBuffer(body)`?

If we do the same thing for ArrayBufferViews as for ArrayBuffer, we would then be copying the view, which we don't want. We need special handling for that (`Buffer.from(body.buffer, body.byteOffset, body.byteLength)`), and I believe that should be in a separate PR.

Reply:
> Why not just `Buffer.from(body)`

No idea that this would just work, but looks like it does, I tested it against [`@octokit/rest`](https://github.com/octokit/rest.js/blob/5a8fca6e69cd5786b58e95933dc789a6dacb2131/test/integration/release-assets-buffer-test.js). Done via f9f328b

> why not use ArrayBuffer.isView(body) instead of isArrayBuffer(body)?

That’s not working, but `body instanceof ArrayBuffer`. Any reason against using that? See 8d3752d

> string-to-arraybuffer 

I’ll leave that as is as per Timothy’s comment